app/viewmodels/ImportVatViewModel.scala

```diff
@@ -0,0 +1,266 @@
+            classes = Some("govuk-summary-list statement-list c79-statements"),
+            id = Some(s"statements-list-$historyIndex")
+          )
+
+          ImportVatCurrentStatementRow(eoriHeading = eoriHeading, dlComponentRow = dlComponentRowContent)
+        } else {
+          ImportVatCurrentStatementRow()
+        }
+      }
+      .filterNot(_.eoriHeading.contains(HtmlFormat.empty))
+
+  private def populateCurrentStatNotAvailableGuidance(implicit messages: Messages): HtmlFormat.Appendable =
+    pComponent("cf.account.vat.no-certificates-available", id = Some("no-certificates-available-text"))
+
+  private def populateCertsOlderThan6MonthsGuidance(serviceUnavailableUrl: Option[String])(implicit
+    messages: Messages
+  ): GuidanceRowWithParagraph =
+    GuidanceRowWithParagraph(
+      h2Heading = h2Component(
+        "cf.account.vat.older-certificates.heading",
+        id = Some("missing-certificates-guidance-heading"),
+        classes = "govuk-heading-m govuk-!-margin-top-9"
+      ),
+      link = Some(
+        linkComponent(
+          "cf.account.vat.older-certificates.description.link",
+          location = serviceUnavailableUrl.getOrElse(emptyString),
+          preLinkMessage = Some("cf.account.vat.older-certificates.description.1")
+        )
+      )
+    )
+
+  private def populateChiefDeclarationGuidance(implicit
+    messages: Messages,
+    appConfig: AppConfig
+  ): GuidanceRowWithParagraph =
+    GuidanceRowWithParagraph(
+      h2Heading = h2Component(
+        "cf.account.vat.chief.heading",
+        id = Some("chief-guidance-heading"),
+        classes = "govuk-heading-m govuk-!-margin-top-6"
+      ),
+      link = Some(
+        linkComponent(
+          appConfig.c79EmailAddress,
+          location = appConfig.c79EmailAddressHref,
+          preLinkMessage = Some("cf.account.vat.older-certificates.description.2")
+        )
+      )
+    )
+
+  private def populateHelpAndSupportGuidance(implicit
+    messages: Messages,
+    appConfig: AppConfig
+  ): GuidanceRowWithParagraph =
+    GuidanceRowWithParagraph(
+      h2Heading = h2Component(
+        id = Some("vat.support.message.heading"),
+        msg = "cf.account.vat.support.heading",
+        classes = "govuk-heading-m govuk-!-margin-top-2"
+      ),
+      paragraph = Some(
+        pComponent(
+          message = "cf.account.vat.support.message",
+          classes = "govuk-body govuk-!-margin-bottom-9",
+          tabLink = Some(
+            hmrcNewTabLinkComponent(
+              NewTabLink(
+                language = Some(messages.lang.toString),
+                classList = Some("govuk-link"),
+                href = Some(appConfig.viewVatAccountSupportLink),
+                text = messages("cf.account.vat.support.link")
+              )
+            )
+          )
+        )
+      )
+    )
+
+  private def populateEoriHeading(certsForAllEoris: Seq[VatCertificatesForEori], historyIndex: Int)(implicit
```

Comment:
do we think its worth grouping the implicit keyword with the following part when its moved onto a new line? like we do on 105?

Reply:
Formatting is done by scala formatter